scripts/docker/Dockerfile

```diff
@@ -0,0 +1,48 @@
+from ubuntu:16.04
+MAINTAINER Nico Heßler <tine20@nico-hessler.de>
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# CONFIGURATION VARIABLES
+ARG TINE20_VERSION=2016.09.5
```

Comment:
Isn't the main reason to have the dockerfile in tree to build whatever is currently checked out? If you are building it from a release zip, it would make more sense to have it out of tree in a different repository.